src/components/Card.tsx

```diff
@@ -122,7 +127,7 @@ const Card = () => {
 					flexShrink: 0,
 					fontFamily: "var(--font-family-outfit)",
 				}}
-				onClick={handleExplainMore}
+				onClick={onExploreMore}
```

Comment:
I cannot see where this function is defined. Not sure if you forgot to add it since the handleExplainMore was initially there but it has now been removed.

Reply:
I'm passing this function now from the parent component
<img width="366" alt="image" src="https://github.com/user-attachments/assets/e7368004-da8f-4c67-96ad-3c3a76be4791">

Reply:
Great! Looks Good.